tox.ini

```diff
@@ -1,8 +1,10 @@
 # content of: tox.ini, put in same dir as setup.py
 [tox]
-envlist = py27
+envlist=py27,py33,py34,py35
+
 [testenv]
+setenv="SIDEBOARD_CONFIG_OVERRIDES=test-defaults.ini"
+deps= -rrequirements.txt
```

Comment:
I don't believe so.  If memory serves, this basically says that we should pass the argument ``-r requirements.txt`` to ``pip``.